thirdeye-ui/src/app/components/subscription-group-wizard/alerts-dimensions/alert-dimension-table/alert-dimension-table.component.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from "@material-ui/core";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    DataGridSelectionModelV1,
+    DataGridV1,
+} from "../../../../platform/components";
+import { DataGridV1Props } from "../../../../platform/components/data-grid-v1/data-grid-v1/data-grid-v1.interfaces";
+import { DimensionRow } from "../add-dimension-dialog/add-dimension-dialog.interface";
+import {
+    areAllRowsSelected,
+    getDataRowsForAlert,
+    getSelectedRows,
+} from "../add-dimension-dialog/add-dimension-dialog.utils";
+import { getAssociationId } from "../alerts-dimensions.utils";
+import { AlertDimensionTableProps } from "./alert-dimension-table.interface";
+
+export const AlertsTable: FunctionComponent<AlertDimensionTableProps> = ({
+    selectedAlert,
+    associations,
+    enumerationItemsForAlert,
+    onChangeAssociations,
+}) => {
+    const { t } = useTranslation();
+
+    const AllDimensionsSelectedRow: DimensionRow = {
```

Comment:
nit pick `AllDimensionsSelectedRow ` => `allDimensionsSelectedRow`

Reply:
Renamed in all instances